data/movie/Willy's Wonderland (2021)/2021-blu-ray-en/disc01.json

```diff
@@ -0,0 +1,206 @@
+{
+  "Index": 1,
+  "Slug": null,
+  "Name": "Willy\u0027s Wonderland",
+  "Format": "Blu-Ray",
+  "Titles": [
+    {
+      "Index": 0,
+      "Comment": "Willy\u0027s Wonderland_t00.mkv",
+      "SourceFile": "00000.mpls",
+      "SegmentMap": "8",
+      "Duration": "1:28:38",
+      "Size": 22178844672,
+      "DisplaySize": "20.6 GB",
+      "Item": null,
```

Comment:
It looks like the items from the summary did not correctly map for some reason. Did you run "finalize" after you created the summary text file? If the mapping worked correctly, the "Item" for the titles that are mapped in the summary file should be populated in the discxx.json file.

Reply:
I probably forgot this. I will go check.